sklearn/preprocessing/_polynomial.py

```diff
@@ -27,6 +32,53 @@
 ]
 
 
+def _append_expansion(X, to_stack, interaction_only, deg, n_features):
+    """Helper function for creating and appending sparse expansion matrices"""
+
+    total_nnz = _calc_total_nnz(X.indptr, interaction_only, deg)
+    expanded_col = _calc_expanded_nnz(n_features, interaction_only, deg)
+
+    if expanded_col == 0:
+        return
+    assert expanded_col > 0
+
+    # This only checks whether each block needs 64bit integers upon
+    # expansion. We prefer to keep 32bit integers where we can,
+    # since currently CSR construction downcasts when possible, so
+    # we'n_features prefer to avoid an unnecessary cast. The dtype may still
+    # change in the concatenation process if needed.
```

Comment:
Are there typos here?
```suggestion
    # This only checks whether each block needs 64bit integers upon
    # expansion. We prefer to keep 32bit integers where we can,
    # since currently SciPy's CSR construction downcasts when possible, 
    # so we prefer to avoid an unnecessary cast. The dtype may still
    # change in the concatenation process if needed.
```